verification/curator-service/ui/src/components/SourceTable.tsx

```diff
@@ -535,21 +534,18 @@ class SourceTable extends React.Component<Props, SourceTableState> {
                             Pagination: (props): JSX.Element => {
                                 return (
                                     <div className={classes.tablePaginationBar}>
-                                        <Typography>
+                                        <Typography style={{ width: '100%' }}>
```

Comment:
hmm shouldn't this be a className instead?

Reply:
I figured it was small enough to use styles, but changed to classes for consistency.